batch/batch/worker/worker.py

```diff
@@ -419,6 +419,9 @@ async def _fuse_mount(self, destination: str, *, credentials_path: str, tmp_path
     async def _fuse_unmount(self, path: str):
         assert CLOUD_WORKER_API
         await CLOUD_WORKER_API.unmount_cloudfuse(path)
+        proc_mounts_stdout, _ = await check_shell_output('cat /proc/mounts')
+        if path in str(proc_mounts_stdout):
+            raise IncompleteCloudFuseCleanup(f'incomplete cloudfuse unmounting: {proc_mounts_stdout}')
```

Comment:
Ah I see now. This looks good, but can we just use python `open` instead of `cat` to read in the contents of `/proc/mounts`?